tests/kola/misc/test-status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash     
+
+# Simple sanity checks for systemd unit status updates.
+
+set -xeuo pipefail
+
+. ${KOLA_EXT_DATA}/libtest.sh
+
+cd $(mktemp -d)
+
+systemctl stop zincati.service
+echo "updates.enabled = false" > /etc/zincati/config.d/disable-in-test.toml
+systemctl start zincati.service
+systemctl status zincati.service | grep 'Status:' > zincati_disabled_status.txt
+assert_file_has_content zincati_disabled_status.txt \
+'initialization complete, auto-updates logic disabled by configuration'
+ok "status show initialization"
+
+systemctl stop zincati.service
+rm /etc/zincati/config.d/*
```

Comment:
This seems a bit aggressive and may have other side-effects. Just overwriting your ow config fragment should be enough.

Reply:
So for example if I use a local cosa build, then it'd have e.g. `90-disable-on-non-production-stream.toml` and `95-disable-on-dev.toml` right?

Reply:
In most cases you will end up with multiple fragments, yes. But only the highest priority setting will matter.

Reply:
Haha thanks! I realized this right after I typed that comment.